packages/botonic-react/src/util/environment.js

```diff
@@ -0,0 +1,27 @@
+export const isDev = process.env.NODE_ENV === 'development'
+export const isProd = process.env.NODE_ENV === 'production'
+
+export const staticAsset = path => {
+  try {
+    const scriptBaseURL = document
+      .querySelector('script[src$="webchat.botonic.js"]')
+      .getAttribute('src')
+    const scriptName = scriptBaseURL.split('/').pop()
+    const basePath = scriptBaseURL.replace('/' + scriptName, '/')
+    return basePath + path
+  } catch (e) {
+    return path
```

Comment:
we should never have silent catches without a comment on why we do it

Reply:
added console.error 👍